File: JavaScript_Basic_04_2023_SolutionsOfProblems/03.Ex_PB-JS-Nested-Conditional-Statements/P05_Journey.js
function journey(input) {
    let budget = Number(input[0]);
    let season = input[1];

    let accommodation = "";
    let destination = "";
    let holidaysCost = 0.0;

    if (budget <= 100) {
        destination = "Bulgaria";
        if (season === "summer") {
            accommodation = "Camp";
            holidaysCost = budget * 0.3;
        } else {
            accommodation = "Hotel";
            holidaysCost = budget * 0.7;
        }
    } else if (budget <= 1000) {
        destination = "Balkans";
        if (season === "summer") {
            accommodation = "Camp";
            holidaysCost = budget * 0.4;
        } else {
            accommodation = "Hotel";
            holidaysCost = budget * 0.8;
        }
    } else {
        destination = "Europe";
        accommodation = "Hotel";
        holidaysCost = budget * 0.9;
    }
    console.log(`Somewhere in ${destination}`);
    console.log(`${accommodation} - ${holidaysCost.toFixed(2)}`);
}

journey(["1500", "summer"])

File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P03Ex_ConditionalStatements_Advanced/P05_Journey.java
package P03Ex_ConditionalStatements_Advanced;

import java.util.Scanner;

public class P05_Journey {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        String season = scanner.nextLine();

        String destination = "";
        String typeRest = "";
        double price = 0;


        if (budget <= 100) {
            destination = "Bulgaria";
            if (season.equals("summer")) {
                price = budget * 0.30;
                typeRest = "Camp";
            } else {
                price = budget * 0.70;
                typeRest = "Hotel";
            }
        } else if (budget <= 1000) {
            destination = "Balkans";
            if (season.equals("summer")) {
                price = budget * 0.40;
                typeRest = "Camp";
            } else {
                price = budget * 0.80;
                typeRest = "Hotel";
            }
        } else {
            destination = "Europe";
            price = budget * 0.90;
            typeRest = "Hotel";
        }
        System.out.printf("Somewhere in %s%n", destination);
        System.out.printf("%s - %.2f", typeRest, price);
    }
}
